roles/jupyterhub/templates/environments/jupyterhub.yaml

```diff
@@ -2,17 +2,18 @@ name: jupyterhub
 channels:
   - conda-forge
 dependencies:
-  - pip
-  - jupyterhub{{ jupyterhub_version }}
-  - oauthenticator
-  - jupyterhub-idle-culler
-  - jupyterhub-systemdspawner
-  - python-keycloak
+  - pip==21.1.2
+  - jupyterhub==2.3.0
```

Comment:
any reason for `{{ jupyterhub_version }}` -> `2.3.0`

Reply:
When I was reviewing the environments to make them work, I checked an old Qhub deployment I had running on that version. No need to keep this one

Reply:
the reason I moved from the env var to hardcoded as for testing, we can have the env var back and just update the version in there

Reply:
oh, I remembered now, it was set to `>=2.3.0` and it was getting the latest which was breaking the other envs, that's why I pinned (moved from var to hardcoded for the reason above)